web/app/js/components/TapLink.jsx

```diff
@@ -24,6 +24,7 @@ const TapLink = ({PrefixedLink, namespace, resource, toNamespace, toResource, pa
 };
 
 TapLink.propTypes = {
+  disabled: PropTypes.bool.isRequired,
```

Comment:
TIOLI: I think `disabled` shouldn't be required; and that it should default to `false` since that'd probably be the most common use case; this doesn't matter much for this instance since `tapLink` is the only place we actually create a `TapLink`....

```
TapLink.defaultProps = {
  disabled: false
}
```

Reply:
no problemo :+1: